mapiproxy/modules/mpm_cache.c

```diff
@@ -386,14 +387,14 @@ static NTSTATUS cache_pull_OpenMessage(struct dcesrv_call_state *dce_call,
 
 	message = talloc((TALLOC_CTX *)mpm, struct mpm_message);
 	NT_STATUS_HAVE_NO_MEMORY(message);
-	
-	message->session = mpm_session_init((TALLOC_CTX *)mpm, dce_call);
+
+	message->session = mpm_session_init(dce_call, NULL);
 	NT_STATUS_HAVE_NO_MEMORY(message->session);
```

Comment:
It is handled there also, as GUID is initialized to zeros, if you try to find by UUID it will not match that one (except for the case where try to find by zeroed UUID...)